### ISSUE TITLE
Particles - Add method for particle object to find parent system

### ISSUE THREAD

**Issue by bobatealee** [Scirra/Construct-feature-requests#372]
### Reviewed guidelines

- [X] I have read and understand the suggestion guidelines

### Checked for duplicate suggestions

- [X] I checked for existing similar suggestions

### Summary

Hi. I am using the particle object feature on Particles to be able to animate individual particles in a system. I am coming across an issue where it is impossible for the particle object to know that it was spawned from a particle system in any way.

This creates issues like not being able to get the parent system's instance variable values, for setting particle object animations based on the parent system's instance variable.

### Possible workarounds or alternatives

There are no workarounds of this that I am aware of. My current method has been making separate particle objects and swapping out the parent system's object based on an instance variable I have set up for the parent system. This is incredibly inefficient.

### Proposed solution

Having a condition equivalent to "Pick parent" for the particle object picking the parent system would be an option. Auto-parenting could also be an option. Realistically, all that needs to happen is establishing a connection between the parent system and particle objects that can be accessed in events.

### Why is this idea important?

This would make using particle objects, especially ones with several animations, much more feasible for projects.

**Comment by F3der1co** [comment]
Agreed, to me the most sense would make a condition on the particle spawner: `pick emitted particle objects` so this is confined to the particle spawner and doesn't bloat other object types

**Comment by bobatealee** [comment]
> Agreed, to me the most sense would make a condition on the particle spawner: `pick emitted particle objects` so this is confined to the particle spawner and doesn't bloat other object types

Super good idea. I've updated the OP with a way that could be further integrated into the Particles plug-in.
